frontend/services/entities/ai_autofill.ts

```diff
@@ -0,0 +1,11 @@
+// import sendRequest from "services";
+import { sendRequest } from "services/mock_service/service/service";
+import endpoints from "utilities/endpoints";
+
+export default {
+  getHumanInterpretationFromSQL: (sql: string): Promise<any> => {
```

Comment:
can we put a type on the response value instead of `Promise<any>`

Reply:
whoops! Thanks for catching this